test-requirements.txt

```diff
@@ -1,6 +1,7 @@
-pytest==5.1.2
-pytest-cov==2.7.1
 coverage==4.5.4
+jsonschema==3.0.2
 pre-commit==1.18.3
+pytest==5.1.2
+pytest-cov==2.7.1
```

Comment:
Could you revert the alphabetic ordering? I'm OK with it, but I'd like to have a separate PR for it.

Reply:
moved everything back. Should I create a separate PR for the rearrange or leave it like this for now?

Reply:
Yes, feel free to open a PR for that!